manual/git-disable-gcm/git-disable-gcm.nuspec

```diff
@@ -3,7 +3,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>git-disable-gcm</id>
-    <title>Disable Git Credential Manager</title>
+    <title>[DEPRECATED] Disable Git Credential Manager</title>
```

Comment:
I still hoped to get this fixed in the Git installer. But maybe its the best to deprecate it and if it once is supported fully by the Git installer we still can support it again.